app/src/main/java/com/hedvig/app/feature/tracking/NotificationOpenedTrackingActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.hedvig.app.feature.tracking
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.hedvig.hanalytics.HAnalytics
+import org.koin.android.ext.android.inject
+
+class NotificationOpenedTrackingActivity : AppCompatActivity() {
```

Comment:
This activity implements the technique called 'reverse activity trampoline' in order to track notification opens in a way that should not be too disruptive for the user.

More reading here: https://onesignal.com/blog/android-12-notification-open-changes/

Reply:
This wont show as a activity transition since we are not creating a view, right?

Reply:
Not as far as I could see - the transition was basically equal to what was in place before.